contracts/Wrapped1155Factory.sol

```diff
@@ -193,7 +225,9 @@ contract Wrapped1155Factory is ERC1155Receiver {
         public
         returns (Wrapped1155)
     {
-        bytes memory deployBytecode = getWrapped1155DeployBytecode(multiToken, tokenId);
+        bytes memory tokenName = hex"577261707065644552432d31313535"; // "WrappedERC-1155";
+        bytes memory tokenNameOffset = hex"000000000000000000000000000000001E";
+        bytes memory deployBytecode = getWrapped1155DeployBytecode(multiToken, tokenId, tokenName, tokenNameOffset);
```

Comment:
Why do we make the `tokenName` dynamic if we hardcode it here?

Reply:
If its just a question of splitting the change into two separate PRs, then I think its a good idea.

Reply:
Sorry, this was removed, it was for testing proposals during the draft PR. The reason was because the `onERC1155Received` and `onERC1155BatchReceived` are override methods from `ERC1155Receiver` interface. So, it was for testing proposals. 

Now we are using `bytes calldata` parameter to pass the token dynamic values, so this is no more necessary.